runtime/parachains/src/builder.rs

```diff
@@ -47,7 +47,7 @@ fn dummy_validation_code() -> ValidationCode {
 /// "features = runtime-benchmarks".
 fn account<AccountId: Decode + Default>(name: &'static str, index: u32, seed: u32) -> AccountId {
 	let entropy = (name, index, seed).using_encoded(sp_io::hashing::blake2_256);
-	AccountId::decode(&mut &entropy[..]).unwrap_or_default()
+	AccountId::decode(&mut &entropy[..]).expect("256 bit input is valid. qed.")
```

Comment:
This is fine for now, but in general it's best to use `TrailZeroInput` to ensure that `AccountId::decode` doesn't run out of input data. In the future, we should just have a trait function for `fn from_entropy(e: impl Encode) -> Self`.